steps/browserstack-upload/0.1.2/step.yml

```diff
@@ -0,0 +1,69 @@
+title: browserstack-upload
+summary: |
+  Upload APK/IPA files to BrowserStack's App Automate
+description: |
+  Upload your build files to Browserstack for remote device testing. Upload your IPA/APK file to BrowserStack App Automation and get back the app's url for your remote device tests.
+website: https://github.com/marcelkalveram/bitrise-step-browserstack-upload
+source_code_url: https://github.com/marcelkalveram/bitrise-step-browserstack-upload
+support_url: https://github.com/marcelkalveram/bitrise-step-browserstack-upload/issues
+published_at: 2018-10-11T17:32:47.851909971+02:00
+source:
+  git: https://github.com/marcelkalveram/bitrise-step-browserstack-upload.git
+  commit: e77a113ffeb57c35255bec81178ad6ac934723a2
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+type_tags:
+- utility
+- test
```

Comment:
I think the test type_tag would be enough. I don't see why we should use the utility tpye_tag for this step.

Reply:
Thanks for the feedback, @BirmacherAkos ! I'll update this.